SpringMVCAnnotationOnlineStore/src/main/java/org/o7planning/springmvconlinestore/dao/impl/OrderDAOImpl.java

```diff
@@ -60,8 +60,8 @@ public void saveOrder(CartInfo cartInfo) {
         int orderNum = this.getMaxOrderNum() + 1;
         Order order = new Order();
  
-        order.setAmount(cartInfo.getQuantityTotal());
-        order.setProdRetailPrice(cartInfo.getAmountTotal()); //Skewed logic - Have Total Field instead?
+        //order.setAmount(cartInfo.getQuantityTotal());
```

Comment:
@KendraMBach, this will need to include shipping cost as well as item cost and quantity. 

Should this also include a sales tax (or is this too complicated)?